lib/middlewares.js

```diff
@@ -142,12 +142,17 @@ exports.fetchEvent = async (req, res, next) => {
     req.approveRequest = approveRequest;
     if (req.approveRequest.body && req.approveRequest.body.success) req.approvePermissions = approveRequest.body.data;
 
+    const limits = await Promise.all(
```

Comment:
this one can be moved to

```
    if (req.user) {
        myApplication = await Application.findOne({
            where: {
                user_id: req.user.id,
                event_id: event.id
            }
        });
    }
```

so it won't fail a request if there's no `req.user`

Reply:
also need to default it to [] if no req.user, so it'd still be an array

Reply:
Yes updated it